auth/src/main/res/layout/fui_phone_layout.xml

```diff
@@ -1,57 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <RelativeLayout
+    <android.support.constraint.ConstraintLayout
         style="@style/FirebaseUI.WrapperStyle"
-        android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
         <com.firebase.ui.auth.ui.phone.CountryListSpinner
             android:id="@+id/country_list"
-            style="@style/FirebaseUI.CountrySpinner" />
+            style="@style/FirebaseUI.CountrySpinner"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintBottom_toBottomOf="@id/phone_layout" />
+
+        <android.support.design.widget.TextInputLayout
+            android:id="@+id/phone_layout"
+            style="@style/FirebaseUI.TextInputLayout.PhoneField"
+            android:layout_width="0dp"
```

Comment:
Is this normal with constraint layout?

Reply:
Yep, it means [`match_constraint`](https://developer.android.com/reference/android/support/constraint/ConstraintLayout.html#DimensionConstraints) which is essentially "go as far as your constraints let you" (or `match_parent` in this case since we don't have any side constraints).